### ISSUE TITLE
About section on profile?

### ISSUE THREAD

**Issue by Qetah** [NamelessMC/Nameless#414]
### NamelessMC About Section

Right now, profile pages aren't extremely customizable on the user's end.
If you look at a similar website software being used by Overcast, you can see that there's an 'about' page listing certain values (social media accounts, interests, etc.) It also includes a large section to write a bio or description of yourself, which supports HTML (basically allowing code-savvy users to create their own Bootstrap mini-page within their profile.)

For reference, see https://oc.tc/anthonny

I figure this wouldn't be too hard to do: just a few text input boxes in the user panel and displays in the 'about' tab, along with the one large HTML box.

**tl;dr I think it'd be a nice, easy-to-code addition to the software as well as a way for users to make their profiles their own.**

**Comment by partydragen** [comment]
much are planned for V2 :P

**Comment by Cl1608Ho** [comment]
I would definitely not allow plain html input as this also allows both html Injection as Javascript injection.

**Comment by Qetah** [comment]
> I would definitely not allow plain html input as this also allows both html Injection as Javascript injection.

Filters could be applied to tags (i.e. no <script>, etc) in order to keep content within the profile section.

**Comment by Cl1608Ho** [comment]
..... It's a bit difficult to keep it filtered if you allow custom styling and such, too. 

Just saying, there are numerous things which aren't just script tags (<img onload=""> or similar)

**Comment by Qetah** [comment]
> ..... It's a bit difficult to keep it filtered if you allow custom styling and such, too. Just saying, there are numerous things which aren't just script tags ( or similar)

That's a fair point, but as shown in the oc.tc example, it's not impossible to do.

**Comment by Cl1608Ho** [comment]
Lol I'm sorry I wrote a tag but github made it invisible. It's img onload os similar what I meant ;)

**Comment by Bayside308** [comment]
Why not allow BB code instead? Much more limited in its capability.

**Comment by Cl1608Ho** [comment]
@Bayside308 yeah, or allow the standard editor. But the whole aim is to provide the user with opportunities to make their profile page their own.

**Comment by samerton** [comment]
Implementation started with https://github.com/NamelessMC/Nameless/commit/bca840ec0b4887e22783d3afae27da760b156d1a, just need to finish displaying the custom fields on profiles.

**Comment by samerton** [comment]
Custom profile fields integrated in v2 pre-release 1 :)
